13_browser.md

```diff
@@ -1,458 +1,263 @@
-((structure)) of the document. The other ((text)) is just plain text.
+标签被包裹在((角括号))(`<` 和 `>`，_小于_ 和 _大于_ 的符号)里，它们提供了关于文档((结构))的信息。其余的文本只是纯文本。
 
 {{index doctype, version}}
 
-The document starts with `<!doctype html>`, which tells the browser to
-interpret the page as _modern_ HTML, as opposed to various dialects
-that were in use in the past.
+该文档以 `<!doctype html>` 作为开头，它告诉浏览器将当前页面视为 _现代_ HTML 来解读，而不是以前被人们使用的诸多“方言”版本。
 
 {{index "head (HTML tag)", "body (HTML tag)", "title (HTML tag)", "h1 (HTML tag)", "p (HTML tag)"}}
 
-HTML documents have a head and a body. The head contains information
-_about_ the document, and the body contains the document itself. In
-this case, the head declares that the title of this document is "My
-home page" and that it uses the UTF-8 encoding, which is a way to
-encode Unicode text as binary data. The document's body contains a
-heading (`<h1>`, meaning "heading 1"—`<h2>` to `<h6>` produce
-subheadings) and two ((paragraph))s (`<p>`).
+HTML 文档有一个头部和主体。头部包含了 _关于_ 文档的信息，而主体包含文档本身。在上述例子中，头部声明了该文档的抬头是“我的主页”，并使用 UTF-8 编码 —— 将 Unicode 文本编码为二进制数据的一种方式。该文档的主体包含了一个标题(`<h1>`，代表“一级标题” —— `<h2>` 至 `<h6>` 可产生不同大小的子标题)以及两个((段落))的 (`<p>`)。
 
 {{index "href attribute", "a (HTML tag)"}}
 
-Tags come in several forms. An ((element)), such as the body, a
-paragraph, or a link, is started by an _((opening tag))_ like `<p>`
-and ended by a _((closing tag))_ like `</p>`. Some opening tags, such
-as the one for the ((link)) (`<a>`), contain extra information in the
-form of `name="value"` pairs. These are called _((attribute))s_. In
-this case, the destination of the link is indicated with
-`href="http://eloquentjavascript.net"`, where `href` stands for
-"hypertext reference".
+标签有好几种不同的形式。一个((元素))，比如主体、段落或者链接都以一个形似 `<p>` 的 _((起始标签))_ 开始，并以一个形似 `</p>` 的 _((闭合标签))_ 结束。有的起始标签以 `名称="值"` 对的形式包含额外的信息，比如用于((链接))的标签(`<a>`)。这些被称为 _((属性))_。在上述示例代码中，该链接的目的地是用 `href="http://eloquentjavascript.net"` 表明的，其中 `href` 意思是 “超文本引用(hypertext reference)”。
 
 {{index "src attribute", "self-closing tag", "img (HTML tag)"}}
 
-Some kinds of ((tag))s do not enclose anything and thus do not need to
-be closed. The metadata tag `<meta charset="utf-8">` is an example of
-this.
+有一些((标签))不包含任何元素，因此无需闭合。元数据标签 `<meta charset="utf-8">` 就是一个这样的范例。
 
 {{index [escaping, "in HTML"]}}
 
-To be able to include ((angle brackets)) in the text of a document,
-even though they have a special meaning in HTML, yet another form of
-special notation has to be introduced. A plain opening angle bracket
-is written as `&lt;` ("less than"), and a closing bracket is written
-as `&gt;` ("greater than"). In HTML, an ampersand (`&`) character
-followed by a name or character code and a semicolon (`;`) is called an _((entity))_
-and will be replaced by the character it encodes.
+为了能够在文档的文本里包含((角括号))，由于它们在 HTML 中有特殊含义，需引入另一种特殊的写法。一个普通的起始角括号写成 `&lt;`("less than")，一个闭合角括号写成 `&gt;` ("greater than")。在 HTML 中，`&` 字符后跟一个名称或字符码、一个分号 (`;`) —— 这样的组合被称为一个 _((实体))_，并且会被其编码的字符给替换掉。
```

Comment:
you should translate "less than" & "greater than" as well

Reply:
Will do.